backend/tests/fixtures/api_client.py

```diff
@@ -0,0 +1,23 @@
+from typing import Callable, Optional
+
+import pytest
+from rest_framework.test import APIClient
+
+from hct_mis_api.apps.account.fixtures import UserFactory
+
+
+@pytest.fixture()
+def unauthenticated_api_client() -> APIClient:
```

Comment:
This name is misleading - it just api client. Where it is used?

Reply:
I used `api_client` name for authenticated client, since it will be used in majority of cases and better for this to be `api_client` than `authenticated_api_client`